spec/models/claim/advocate_claim_spec.rb

```diff
@@ -53,17 +53,20 @@
   end
 
   describe 'validate external user is advocate role' do
-    let(:claim) { build(:unpersisted_claim, :with_fixed_fee_case) }
+    subject { claim.external_user.is?(:advocate) }
```

Comment:
I presume that it is OK that the claim that is being tested is now created by a different factory.

Reply:
I think so. The `advocate_claim` and `unpersisted_claim` factories both create a claim with an `external_user` with the `advocate` role, which is all this test needs.